.zuul.yaml

```diff
@@ -0,0 +1,27 @@
+---
+- project:
+    name: openstack-k8s-operators/test-operator
+    github-check:
+      jobs:
+        - podified-multinode-edpm-deployment-crc:
+            vars:
+              test_fw: test_operator
```

Comment:
let's set this var:
cifmw_test_operator_tempest_include_list
and run all tests? so that we see what fails